### DISCUSSION TITLE
When I compile, a `[MemberName]` code inspection warning causes the compilation to fail

### DISCUSSION THREAD

**Discussion by zenvzenv** [open-telemetry/opentelemetry-java-instrumentation#9449] [Q&A] [answered]
I added a new instrumentation module,but the instrumented server or library jar isn't available in any public Maven repository,so I reate a module with stub classes that define only the methods that I need to write the instrumentation.I follow the [Instrumenting code that is not available as a Maven dependency](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/contributing/writing-instrumentation.md#instrumenting-code-that-is-not-available-as-a-maven-dependency) content create a `compile-stub` module.The instrumented library has a method named `requestURI()`, so I add `requestURI()` in `compile-stub`,but when I compile the project,there a waring lead to failed.

```text
E:\myworkspace\opentelemetry-java-instrumentation-1.29.0\instrumentation\tongweb\compile-stub\src\main\java\com\tongweb\connector\Request.java:32: Warn: [MemberName] Methods and non-static variables sho
uld be named in lowerCamelCase, with acronyms treated as words (https://google.github.io/styleguide/javaguide.html#s5.3-camel-case); did you mean 'requestUri'?
  public MessageBytes requestURI() {
                      ^
```

Now I commented `-Werror` in the `otel.javaagent-bootstrap.gradle.kts` file,so compile success.But this is just a stopgap, I would like to know how to exclude the compile-stub module from code inspection, or what are the best practices

**Comment by laurit** [comment, marked as answer]
Try adding `@SuppressWarnings("MemberName")` to the class. After that you'll probably get the next error from checkstyle, that you can suppress with https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/dc523cf9902cd606ff293331bd53392d2b284f23/instrumentation/liberty/compile-stub/build.gradle.kts#L11-L15
